src/check/arbitrary/ObjectArbitrary.ts

```diff
@@ -491,37 +441,12 @@ function unicodeJsonObject(constraints?: number | JsonSharedConstraints): Arbitr
  *
  * Keys and string values rely on {@link string}
  *
- * @remarks Since 0.0.7
- * @public
- */
-function json(): Arbitrary<string>;
-/**
- * For any JSON strings with a maximal depth
- *
- * Keys and string values rely on {@link string}
- *
- * @param maxDepth - Maximal depth of the generated objects
- *
- * @deprecated
- * Superceded by `fc.json({maxDepth})` - see {@link https://github.com/dubzzz/fast-check/issues/992 | #992}.
- * Ease the migration with {@link https://github.com/dubzzz/fast-check/tree/main/codemods/unify-signatures | our codemod script}.
+ * @param constraints - Constraints to be applied onto the generated instance (since 2.5.0)
  *
  * @remarks Since 0.0.7
  * @public
  */
-function json(maxDepth: number): Arbitrary<string>;
-/**
- * For any JSON strings
- *
- * Keys and string values rely on {@link string}
- *
- * @param constraints - Constraints to be applied onto the generated instance
- *
- * @remarks Since 2.5.0
- * @public
- */
-function json(constraints: JsonSharedConstraints): Arbitrary<unknown>;
-function json(constraints?: number | JsonSharedConstraints): Arbitrary<string> {
+function json(constraints: JsonSharedConstraints = {}): Arbitrary<unknown> {
```

Comment:
```suggestion
function json(constraints: JsonSharedConstraints = {}): Arbitrary<string> {
```

By the way, there is a typo there